package.json

```diff
@@ -30,4 +30,4 @@
 		"typescript": "^5.0.0"
 	},
 	"prettier": "prettier-config-rschristian"
-}
\ No newline at end of file
+}
```

Comment:
Please revert this & the deleted `yarn.lock`

Reply:
Sorry, I accidentally deleted it during testing.

Reply:
No worries

Reply:
fixed, https://github.com/preactjs/create-preact/pull/16/commits/b766b92d97a4e8f044fc27ab504f7b59533c19bd